chroma-manager/tests/framework/integration/installation_and_upgrade/jenkins_steps/main

```diff
@@ -6,7 +6,7 @@
 set_defaults true
 check_for_autopass
 
-export CLUSTER_CONFIG_TEMPLATE=${CLUSTER_CONFIG_TEMPLATE:-"chroma/chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/shared_storage_configuration_cluster_cfg.json"}
+export CLUSTER_CONFIG_TEMPLATE=${CLUSTER_CONFIG_TEMPLATE:-"$CHROMA_DIR/chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/shared_storage_configuration_cluster_cfg.json"}
```

Comment:
So, for the rest of this test, it happens [at](https://github.com/intel-hpdd/intel-manager-for-lustre/commit/de4e8dbd3974d8ad4ef357a8afc1f9fa295ddccc#diff-86e1d83b074882382896e9e2bff4065eR22) but that doesn't help here.

Typically it's set by the test caller, like we do in our Jenkins steps, but it should have a reasonable default set if the caller doesn't set it, to reduce the amount the caller is supposed to know about how the tests run.  I'll push another patch closing up this and any other loophole like this.